### PR TITLE
Release v3.9.3

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -22,9 +22,9 @@ Bug fixes
 
   *Related issues and pull requests on GitHub:*
   :issue:`8097`, :issue:`8098`.
-  
-  
-
```
Comment (Dreamsorcerer) [inline CHANGES.rst]: @webknjaz Atleast pre-commit fixes it automatically when I forget.

**Comment by Dreamsorcerer** [comment]
So weird, that's the second time the 3.12 macos job has just cancelled for no apparent reason...

File: CHANGES.rst (hunk 1 above)
Comment (webknjaz) [reply CHANGES.rst]: Ah, good to know.

**Comment by webknjaz** [comment]
> So weird, that's the second time the 3.12 macos job has just cancelled for no apparent reason...

> [Test (macos, 3.12, true, Y)](https://github.com/aio-libs/aiohttp/actions/runs/7700647179/job/20984945691)
GitHub Actions has encountered an internal error when running your job.

@Dreamsorcerer this happens when there's issues within the GitHub platform or their CI runners.

**Pull request merged**